.github/workflows/test-changed-auth.yml

```diff
@@ -16,7 +16,9 @@ jobs:
       - name: install Chrome stable
         run: |
           sudo apt-get update
-          sudo apt-get install google-chrome-stable
+          apt-get install wget
```

Comment:
you need sudo here too. `sudo apt-get install wget`

Reply:
Done. Thanks!